mappers/tva.js

```diff
@@ -29,7 +29,7 @@ function mapTvaContact(recordId, record) {
     if (title) {
       statements.push(new Statement(sym(uri), VCARD('honorific-prefix'), lit(title, 'nl')));
     } else {
-      console.error(`Cannot map TVA contact title value '${record['tva_contact_title']}' for record ${recordId}`);
+      errorLogger('TVA contact title', record['tva_contact_title'], recordId);
```

Comment:
Pass the same field name as used to lookup the value as first argument. I know it was different before, but old code is wrong.